demo/src/main/scala/demo/AladinContainer.scala

```diff
@@ -191,7 +193,11 @@ object AladinContainer {
           r.backend.addCatalog(cat) *>
           updateSvgState.flatMap { s =>
             r.backend.recalculateView *> r.backend.runOnAladinCB(updateVisualization(s))
-          }
+          } *>
+          r.backend.world2pixFn
+            .flatMap(f => Callback.log(s"${f(testCoord).get._1}x${f(testCoord).get._2}")) *>
+          r.backend.pix2worldFn
+            .flatMap(f => Callback.log(s"${f(48742, 49134).get}"))
```

Comment:
Did you want to keep this here?

Reply:
Yeah. this is for the demo, essentially this proves me that the functions are doing the right thing